python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -572,20 +597,25 @@ async def do_resetOffset(self, data):
         # best to do this immediately and not later in a loop as it'll be confusing to see
         # large offsets pop up when the loop comes on rather then when the reset command is set
         if 'atspectrograph' in self.corrections:
-            offset_to_apply = (
-                self.focus_offset_per_category['filter'] + self.focus_offset_per_category[
-                    'disperser'] + self.focus_offset_per_category['wavelength']
+            # Flake8 giving error W503 about a line break before a binary operator. However, it also
```

Comment:
Please, remove this comment about flake8 - W503. We adopted this exception some time ago and the ATAOS was not updated. You can add this information to the commit message though.